### ISSUE TITLE
May I know how to get the max_drawdown start/end time?

### ISSUE THREAD

**Issue by richie-liu** [edtechre/pybroker#163]
May I know how to get the max_drawdown start/end time?

I try to get max_drawdown starat/end time, but I did not find any clue on document

**Comment by edtechre** [comment]
Hi @richie-liu,

The bar that the max_drawdown occurred at is not recorded in the [EvalMetrics](https://www.pybroker.com/en/latest/reference/pybroker.eval.html#pybroker.eval.EvalMetrics), but I can add it.

**Comment by richie-liu** [comment]
yes,I think so. it could help parse reason why it is max drawdown  . Thanks :)

**Comment by edtechre** [comment]
Added in a06ab95 in dev.
